include/stdio.h

```diff
@@ -192,10 +192,11 @@ int    vfprintf(FAR FILE *stream, FAR const IPTR char *fmt,
          va_list ap);
 int    vfscanf(FAR FILE *stream, FAR const IPTR char *fmt, va_list ap);
 int    vprintf(FAR const IPTR char *fmt, va_list ap);
+int    vscanf(FAR const IPTR char *fmt, va_list ap);
```

Comment:
These changes seem out of place in the PR.  Can we move this to a different PR.  We should not make any IPTR-related changes until we have a clear understanding and a plan for how we want to handle these.